examples/hosts/app-integration/external-controller/README.md

```diff
@@ -80,5 +79,5 @@ const connectionConfig: AzureConnectionConfig = useAzure ? {
 ```
 
 In this way, we can toggle between remote and local mode using the same config format. We make use of
-`AzureFunctionTokenProvider` for running against live FRS instance since it is more secured, without exposing the tenant
+`AzureFunctionTokenProvider` for running against live Azure Fluid Relay  instance since it is more secured, without exposing the tenant
```

Comment:
```suggestion
`AzureFunctionTokenProvider` for running against live Azure Fluid Relay instance since it is more secured, without exposing the tenant
```